images/linux/scripts/installers/1804/basic.sh

```diff
@@ -133,6 +133,9 @@ apt-get install -y --no-install-recommends parallel
 echo "Install gnupg2"
 apt-get install -y --no-install-recommends gnupg2
 
+echo "Install zlib"
+apt-get install -y --no-install-recommends lib32z1
```

Comment:
should we be adding the 64 bit packages as well? otherwise it seems incomplete

Reply:
@alepauly Thank you for your review! It's only required to have this one to solve the issue mentioned in the description and furthermore size of lib64z1 is about 12 MB. So could you please confirm that it's should be installed despite its size?

Reply:
@alepauly , I think it is okay to keep only `lib32z1`. It has size 120 KB.
`lib64z1` has version 12 MB. So if it is not required for this particular issue it doesn't make sense to bring both.